src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertFrameworks.sql

```diff
@@ -1,4 +1,21 @@
-﻿IF NOT EXISTS (SELECT * FROM dbo.Framework)
-    INSERT INTO dbo.Framework(Id, [Name], [Owner])
-    VALUES ('NHSDGP001', 'NHS Digital GP IT Futures Framework 1', 'NHS Digital');
+﻿DECLARE @frameworks AS TABLE
+(
+    [Id]          NVARCHAR (10)  NOT NULL PRIMARY KEY,
+    [Name]        NVARCHAR (100) NOT NULL,
+    [Owner]       NVARCHAR (100) NULL
+);
```

Comment:
Can you update the formatting of the table variable to look like the below?

```sql
DECLARE @frameworks AS TABLE
(
    Id nvarchar(10) NOT NULL PRIMARY KEY,
    [Name] nvarchar(100) NOT NULL,
    [Owner] nvarchar(100) NULL
);
```

(We usually use lower case for data types and tend not to align the column names and data types. We only use square brackets where required too.)

Reply:
Formatted table query